modules/benchmark/R/calc.benchmark.R

```diff
@@ -18,28 +18,31 @@ calc.benchmark <- function(settings, con){ #settings file is output from start.b
   
   # Query BRR for benchmark ids
   
-  bm.id <- settings.bm$run$id #maybe something like that?
+  bm.id <- settings$benchmark$id #maybe something like that?
   
   # For each bm.id
   #  Query database for: input path, variables, site, start/end dates, metrics, formats
   
-  input.path <- db.query.file.path(input.id,host_name,con)
-  f_v <- db.query.format.vars(input.id,con)
-  format_table <- f_v[[1]]
-  vars_names_units <- f_v[[2]]
+  data.path <- query.file.path(bm.id,settings$run$host$name,con)
+  format <- query.format.vars(bm.id,con)  
   
   
   # Need to do something here to narrow down the variables to be used
   # rows <- which(vars_names_units$bety_name %in% calc.vars)
   # vars_names_units <- vars_names_units[rows,]
   
-  site  <- db.query.site(input.id, con)
+  site  <- query.site(settings$run$site$id, con)
   
-  metrics <- db.query(paste("SELECT m.id, m.name from metrics as m JOIN benchmarks_metrics as b 
-                            ON m.id = b.metric_id WHERE b.benchmark_id = ", bm.id),con)
+  metrics <- db.query(paste("SELECT m.name from metrics as m JOIN benchmarks_metrics as b 
+                              ON m.id = b.metric_id WHERE b.benchmark_id = ", bm.id),con)
+  
+  model_run <- dir(settings$run$host$outdir, full.names=TRUE)[1]
+  #How are we dealing with ensemble runs? Right now I've hardcoded to select the first run.
```

Comment:
That's fine for now since that's the median. We'll have to generalize this fairly soon to detect the ensemble size and load all ensemble members (cause the longer we wait to generalize, the more painful it will be, but this can wait till after the May meeting)